docs/pages/blog/docs-updates.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Our docs just got a major upgrade—here's what that means for you'
+description: Each of MUI's products now has its own dedicated documentation, making it easier than ever to find exactly what you need.
+date: 2022-04-06T00:00:00.000Z
+authors: ['danilo-leal']
+tags: ['News']
+---
+
+As MUI continues to grow beyond our flagship product, Material UI (we [rebranded the company](/blog/material-ui-is-now-mui/) last year as a first step), it has become clear that the documentation for our products can no longer all live under one roof.
+
+That's why we're excited to announce that we are shipping a major upgrade to our documentation, to make it easier than ever to find exactly what you need—no matter which MUI products you're working with.
+
+## Wait... what MUI products are you talking about?
+
+We currently offer two main product lines:
+
+- **MUI Core**—a collection of foundational component libraries, including:
+  - _Material UI_ —components that implement Google's Material Design
+  - _MUI Base_ —unstyled components for implementing your own design system
+  - _MUI System_ —CSS utilities for quickly laying out design systems
+- **MUI X**—a collection of advanced components for complex use cases, including:
+  - _Data Grid_ —fast, feature-rich, extendable React data table
+  - _Date and Time Pickers_ —interface control components for selecting dates and times
+
+## What has changed?
+
+All MUI products still live under the [mui.com](http://mui.com) domain, but each of them now has its own respective URL and documentation.
+We added an identifier and menu to the upper-left corner of the docs for improved navigation:
+
+<img src="/static/blog/docs-updates/docs-separation.png" style="width: 796px; margin-top: 16px; margin-bottom: 16px;" alt="Screenshot of the product identifier menu" />
+
+As for the URLs, this is how they look now:
+
+- MUI Core:
+  - Material UI: [https://mui.com/material-ui/](https://mui.com/material-ui/getting-started/installation/)
+  - MUI Base: [https://mui.com/base/](https://mui.com/base/getting-started/installation/)
+  - MUI System: [https://mui.com/system/](https://mui.com/system/basics/)
+- MUI X:
+  - Data Grid: [https://mui.com/x/react-data-grid/](https://mui.com/x/react-data-grid/)
+  - Date and Time Pickers: [https://mui.com/x/react-date-pickers/](https://mui.com/x/react-date-pickers/getting-started/)
+
+> 📖 The MUI X date and time pickers has been moved from the lab (`@mui/lab`). If you are curious about it, check out the [dedicated blog post](/blog/lab-date-pickers-to-mui-x/).
+
+### Improved search experience
+
+The documentation restructuring ranks search results based on the product that you are currently looking at.
+For example, if you are looking at Material UI documentation, when you press <kbd>⌘ K</kbd> (<kbd>Ctrl + K</kbd> on Windows) and type a keyword, you will find that most of the results are related to Material UI.
```

Comment:
The standard syntax and a11y one too. Same as done in https://mui.com/components/data-grid/accessibility/#navigation. AFAIK It **must** be:

```suggestion
For example, if you are looking at Material UI documentation, when you press <kbd><kbd class="key">⌘</kbd>+<kbd class="key">K</kbd></kbd> (<kbd><kbd class="key">CTRL</kbd>+<kbd class="key">K</kbd></kbd> on Windows) and type a keyword, you will find that most of the results are related to Material UI.
```